content/02.author-sections.md

```diff
@@ -441,7 +439,40 @@ This is an improved version of my main figure in this interesting study [@doi:10
 
 
 <!-- Binglan Li -->
+### Binglan Li
+Greeting from the [Ritchie Lab](http://ritchielab.org/) at the University of Pennsylvania. I am a third year graduate student in the Genomics and Computational Biology programe and interested in prioritization of drug response-related gene via data integration approaches. 
+
+I am still on the early part of research journey. But I would love to share my latest work published in the PSB 2019 proceedings 2019.
+- Influence of tissue context on gene prioritization for predicted transcriptome-wide association studies [
+@doi.org/10.1142/9789813279827_0027]
```

Comment:
Move `@doi.org/10.1142/9789813279827_0027]` to the previous line so that `[@doi.org/10.1142/9789813279827_0027].` is all on the same line.